src/components/RequestList/RequestItem/index.tsx

```diff
@@ -163,10 +162,15 @@ const RequestItem: React.FC<RequestItemProps> = ({
                 {isMovie(title) ? title.title : title.name}
               </a>
             </Link>
-            <div className="text-sm">
-              {intl.formatMessage(messages.requestedby, {
-                username: requestData.requestedBy.displayName,
-              })}
+            <div className="flex items-center">
+              <img
+                src={requestData.requestedBy.avatar}
+                alt=""
+                className="w-4 mr-1 rounded-full sm:mr-2 sm:w-5"
```

Comment:
Do we need responsive sizes here?

Reply:
Ah you're right, we don't!  I just copied what you had from the request card, haha.  I'll remove that :)